.travis.yml

```
@@ -18,7 +18,7 @@ before_install:
   - conda config --add channels conda-forge
   - conda update -q conda
 install:
-  - conda create -q -n test-env python=$TRAVIS_PYTHON_VERSION numpy cython "ipopt<3.13" future six setuptools sphinx numpydoc pkg-config
+  - conda create -q -n test-env python=$TRAVIS_PYTHON_VERSION numpy cython ipopt future six setuptools sphinx numpydoc pkg-config
   - source activate test-env
```

```suggestion
  - conda activate test-env
```
We may want to give it a try as source activate seems to be considered outdated.